nservicebus/operations/metrics/service-control.md

```diff
@@ -24,16 +24,26 @@ snippet: SendMetricDataToServiceControl
 
 ### Service Control Metrics Address
 
-The default is `particular.monitoring` which is ServiceControl monitoring instance input queue.
+The default instance name is `particular.monitoring` which is also used as the input queue for ServiceControl Monitoring.
 
 partial: interval
 
 ### Instance Id
 
 An override for `$.diagnostics.hostid` and `$.diagnostics.hostdisplayname`.
 
-By default the monitoring plug-in will use [host identifiers](/nservicebus/hosting/override-hostid.md) to identify the monitored endpoint instance.
+This value is shown in the list of instances on the endpoint details page in ServicePulse Monitoring. By default the monitoring plug-in will use the [NServiceBushost host id hash identifier](/nservicebus/hosting/override-hostid.md) to identify the monitored endpoint instance.
```

Comment:
This needs to be changed after https://github.com/Particular/LaunchWave.DevOpsAdvancedMonitoring/issues/438 has been fixed.

So it should be by: By default it will show HostId which is guid. One can override it with HostDisplayName. InsanceId should not be used unless using Azure CloudHost.